closed/custom/gensrc/GensrcVarHandles-post.gmk

```diff
@@ -1,22 +0,0 @@
-# ===========================================================================
-# (c) Copyright IBM Corp. 2018, 2020 All Rights Reserved
-# ===========================================================================
-# This code is free software; you can redistribute it and/or modify it
-# under the terms of the GNU General Public License version 2 only, as
-# published by the Free Software Foundation.
-#
-# IBM designates this particular file as subject to the "Classpath" exception
-# as provided by IBM in the LICENSE file that accompanied this code.
-#
-# This code is distributed in the hope that it will be useful, but WITHOUT
-# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
-# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
-# version 2 for more details (a copy is included in the LICENSE file that
-# accompanied this code).
-#
-# You should have received a copy of the GNU General Public License version
-# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
-# ===========================================================================
-
-# OpenJ9 uses it's own VarHandles implementation
-GENSRC_JAVA_BASE := $(filter-out $(GENSRC_VARHANDLES),$(GENSRC_JAVA_BASE))
```

Comment:
This file cannot be deleted; instead this must be conditional so builds without `--enable-openjdk-methodhandles` can work:
```
ifneq (true,$(OPENJ9_ENABLE_OPENJDK_METHODHANDLES))
  # OpenJ9 uses it's own VarHandles implementation
  GENSRC_JAVA_BASE := $(filter-out $(GENSRC_VARHANDLES),$(GENSRC_JAVA_BASE))
endif # OPENJ9_ENABLE_OPENJDK_METHODHANDLES
```

Reply:
Addressed.